xonsh/execer.py

```diff
@@ -8,7 +8,7 @@
 import warnings
 from collections import Mapping
 
-from xonsh import ast
+from xonsh.ast import CtxAwareTransformer
```

Comment:
Yes, so this is actually correct.  The way the code amalgamater works is the other moduled that are in the *same* package (and amalgamated) should be `from pkg.x import a, c, d`. This is because the amalgamater puts all such modules in the same globals(), which is effectively what the from-imports do.

So since `xonsh.ast` and `xonsh.execer` are both in the same poackage (`xonsh`) they should have from-imports. 

Alternatively, for modules outside of the current package (or modules that are not amalgamated) the import for be either `import pkg.x` or `import pkg.x as name`. This is because these are the only cases where automatically inserting lazy imports in guarantted to be safe. This is because `from pkg.x import name` may import a variable that cannot be lazily constructed.